pkg/debug/scripts/collect-info.sh

```diff
@@ -45,6 +58,30 @@ while getopts "vh" o; do
     esac
 done
 
+# We are not on EVE? Switch to read-logs mode
+if [ ! -d "/persist" ]; then
```

Comment:
Hmm - I have a /persist on my laptop.

Can we avoid making this assymption? E.g., by adding an -e (for extract) or -A (for All) for the case where you currently look for /persist?

Reply:
Let me do the opposite: 
```
# We are not on EVE? Switch to read-logs mode
if [ -d "$SCRIPT_DIR/persist-newlog" ]; then
      ...
fi
```

The script is the part of the tarball. So the debugging procedure is the following:
```
tar -xf eve-info-v8-2023-09-27-08-45-19.tar.gz
./eve-info-v8-2023-09-27-08-45-19/collect-info.sh > all-logs.json
```

I would like to avoid any mode options, because the only thing you can do once the tarball is collected is to extract all the logs, collecting logs makes sense only when the script is on EVE. So there is no actually an option, no choice, the action is determined by the script placement. Ideally there should be another script, but I do not like this zoo of scripts as well.